### ISSUE TITLE
UI: Left Sidebar for organization management should remain in view when scrolling long content

### ISSUE THREAD

**Issue by jaaydenh** [coder/coder#14420]
When scrolling long content for tables on the right side of the view such as Groups, Members and Roles, the left sidebar should remain fixed in view.

<img width="312" alt="Screenshot 2024-08-23 at 12 00 43 PM" src="https://github.com/user-attachments/assets/405146d6-3f48-45fb-897c-2ab7aa5abb58">

**Comment by chrifro** [comment]
I agree. I highlighted as one of the changes in https://github.com/orgs/coder/projects/15/views/35?pane=issue&itemId=79922411&issue=coder%7Ccoder%7C14654
